integration_tests/scripts/ituploader.js

```diff
@@ -0,0 +1,75 @@
+const {ethers, Contract} = require("ethers");
+const crypto = require('crypto');
+const {EthStorage} = require("ethstorage-sdk");
+const core = require('@actions/core');
+const fs = require('fs');
+
+const dotenv = require("dotenv")
+dotenv.config()
+const privateKey = process.env.ES_NODE_UPLOADER_PRIVATE_KEY;
+const contractAddr = process.env.ES_NODE_CONTRACT_ADDRESS;
+const RPC = 'http://65.109.20.29:8545';
+const contractABI = [
+    "function lastKvIdx() public view returns (uint40)"
+]
+
+const provider = new ethers.JsonRpcProvider(RPC);
+const contract = new Contract(contractAddr, contractABI, provider);
+const MAX_BLOB = 256n;
```

Comment:
The test shard contains 32k blobs, and if we only upload 256 blobs, it will be very difficult during the mining process to hit the key-value pairs that contain actual data and submit a proof that includes them. This means we might miss a very important case.

Reply:
Change to 4096, as each submission will sample 8 times, so every submission will have chances to cover blobs that are not empty.

Reply:
I think we should check whether we have sampled the key-value pairs that contain the actual data

Reply:
after we support multi-shard, we should fill in all shard 0, so the sampling will cover the key-value pairs.

Reply:
add TODO for this